notebooks/diagnostics/pop/basics_pop.ipynb

```diff
@@ -212,16 +224,18 @@
    "source": [
     "# Set your username here:\n",
     "username = \"PUT_USER_NAME_HERE\"\n",
+    "casename = 'b.day2.1'\n",
     "\n",
     "# Here we point to the archive directory from your b.day2.1 simulation\n",
-    "pth = f\"/glade/scratch/{username}/archive/b.day2.1/ocn/hist/\"\n",
+    "pth = f\"/glade/scratch/{username}/archive/' + casename + '/ocn/hist/\"\n",
     "\n",
     "# If you were unable to successfully run the b.day2.1 simulation, then feel free to use\n",
     "# this provided simulation data instead:\n",
-    "#pth = \"/glade/p/cesm/tutorial/tutorial_2023_archive/b.day2.1/ocn/hist\"\n",
+    "#pth = '/glade/p/cesm/tutorial/tutorial_2023_archive/' + casename + '/ocn/hist/'\n",
     "\n",
-    "# You can also choose to examine output from a longer POP2 run here:\n",
-    "#pth = '/glade/scratch/gmarques/archive/gpop.e23.GJRA.TL319/ocn/hist/'\n",
+    "# You can also choose to examine output from a longer POP2 run here by uncommenting both lines below:\n",
+    "#casename = 'gpop.e23.GJRA.TL319'\n",
```

Comment:
I think we should remove this because the data will be purged in a few months.

Reply:
Done, removed these paths.